releases/Process/Stabilization Process.md

```diff
@@ -0,0 +1,85 @@
+# Draft - O3DE Stabilization Process
+
+O3DE Stabilization is the process we use to prepare the codebase for a release. During Stabilization we create a branch off of the development branch to create a “stabilization branch”, limit any new feature introduced into the branch, and the community fixes bugs. This helps to ensure O3DE releases do not ship with bugs that will block a user’s ability to utilize O3DE. Note that stabilization is not mean to catch every issue, and instead works in conjunction with unit tests, nightly automated test runs, functional automated testing, manually running test passes, and issues filed by the community.  
+  
+
+## Key Details
+
+-   The Stabilization Process typically takes place as part of the Release Process. Please refer to the [O3DE Major Release Process](https://github.com/o3de/sig-release/blob/main/releases/Process/Major%20Release%20Process.md) for more information about O3DE releases.
+-   Stabilization involves the following O3DE Repositories:
+    -   o3de
+    -   o3de-multiplayersample
+    -   o3de-netsoaktest
+
+## Roles and Responsibilities
+*Intent: Outlines the minimum set of roles and their responsibilities in order to complete all of the necessary release tasks.*
+
+The following outlines roles and responsibilities related to the Stabilization process. This list represents a subset of the roles involved in the O3DE Release. The other roles involved in the release are identified in the [O3DE Major Release Process](https://github.com/o3de/sig-release/blob/main/releases/Process/Major%20Release%20Process.md) 
+
+|SIG or Group|Title|Count|Description
+|--|--|--|--
+Build SIG| Release Deployment Engineer|1|Responsible for creating the stabilization branch, uploading the release, tagging Release, and publishing the version number update to o3debinaries.org . Must have maintainer privileges.
+|Any SIG|Stabilization Integrator|up to 12|During the stabilization phase, these community members are responsible for performing "branch maintenance" by merging code from the stabilization branch to the development branch according to a schedule. This branch maintenance is ensures that all of the code submitted into the stabilization branch is also applied back to the development branch. Integrators must have maintainer privileges.
```

Comment:
To confirm my understanding, by any SIG do you mean all SIGs should have someone as Stabilization Integrator?

Reply:
All sigs that contribute code.

Reply:
Got it. When I read "any SIG", I thought only one of the SIGs will become the integrator. I propose the word to be "All SIGs" instead. Thoughts?

Reply:
Agree this needs changing. Maybe I'll Change to "All Code Contributing SIG*" and clarify the list of eligible SIG.

Reply:
address in next commit

Reply:
Weird, I don't see any changes in the PR regarding this. Did you push your changes into the branch? @tonybalandiuk